docs/source/deploy_chaincode.md

```diff
@@ -0,0 +1,348 @@
+
+After a sufficient number of organizations have approved a chaincode definition, one organization can commit the chaincode definition to the channel. If a majority of channel members have approved have approved the same chaincode definition, the commit transaction will be successful and the parameters agreed in the chaincode definition will be implemented on the channel.
+
+You can use the `peer lifecycle chaincode checkcommitreadiness` command to check whether channel members have approved the same chaincode definition. Use the following command to check if Org1 and Org2 have approved the chaincode definition for Fabcar on the channel `mychannel`. The flags used for the `checkcommitreadiness` command are identical to the flags used to approve a chaincode for your organization. However, you do not need to include the `--package-id` flag.
+```
+peer lifecycle chaincode checkcommitreadiness --channelID mychannel --name fabcar --version 1.0 --init-required --sequence 1 --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --output json
+```
+
+The command will produce a JSON map that display if a channel member member has approved the parameters that were specified in the `checkcommitreadiness` command:
+```json
+    {
+            "Approvals": {
+                    "Org1MSP": true,
+                    "Org2MSP": true
+            }
+    }
+```
+
+Since both channel members have approved the same parameters, the chaincode definition can be committed to the channel. Issue the following command to commit the chaincode definition to the channel as the Org2 admin:
+```
+peer lifecycle chaincode commit -o localhost:7050 --ordererTLSHostnameOverride orderer.example.com --channelID mychannel --name fabcar --version 1.0 --sequence 1 --init-required --tls true --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+```
+
+The commit command also needs to be submitted by an organization admin. Note that the transaction includes the address of the ordering node and uses the `--peerAddresses` flag to target peers in Org1 and Org2. The `commit` transaction needs to query the peers of channel members that have approved the chaincode definition. The transaction is then submitted to the ordering service to be added to a block and distributed to the channel, where the peers validate that a majority of organizations have approved the chaincode definition. The `peer lifecycle chaincode commit` command will wait for the validations from the peer before returning a response.
```

Comment:
Is it a requirement that you have to target the peers that approved the cc definition?
It's unclear to me what peer addresses need to be specified in the --peerAddresses flag?

Reply:
Changed to "The commit command also needs to be submitted by an organization admin. The transaction above uses the `--peerAddresses` flag to target `peer0.org1.example.com` from Org1 and `peer0.org2.example.com` from Org2. The `commit` transaction is submitted to the peers of channel members to query the chaincode definition approved by the peer's organization. The command needs to target a sufficient number of peers to meet the policy for deploying a chaincode. Because the approval is distributed within each organization, you can target any peer from each organization. "